sktime/datatypes/_utilities.py

```diff
@@ -331,33 +333,35 @@ def get_slice(obj, start=None, end=None):
 
     obj = convert_to(obj, GET_LATEST_WINDOW_SUPPORTED_MTYPES)
 
-    if (
-        isinstance(start, pd.Timestamp)
-        or isinstance(end, pd.Timestamp)
-        or isinstance(start, pd.Period)
-        or isinstance(end, pd.Period)
-    ):
+    # numpy3D (Panel) or np.npdarray (Series)
+    # Assumes the index is integer so will be exclusive by default
+    if isinstance(obj, np.ndarray):
```

Comment:
hm, I think this is not quite right.

`numpy3D` (for a panel of series) has index (instance, variable, time), so you need to subset the *third* index in that case.
`np.ndarray` has (time, variable) indexing.

I know, this is strangely inconsistent, but it is individually consistent with community preference and the `tslearn` package for time series classification.

Reply:
Looks good